checkov/terraform/graph_builder/utils.py

```diff
@@ -317,3 +324,15 @@ def setup_file_path_to_referred_id(graph_object: DiGraph | igraph.Graph) -> dict
         return get_file_path_to_referred_id_igraph(graph_object)
     else:  # the default value of the graph framework is 'NETWORKX'
         return get_file_path_to_referred_id_networkx(graph_object)
+
+
+def get_attribute_is_leaf(vertex: TerraformBlock) -> Dict[str, bool]:
+    attribute_is_leaf = {}
+    attribute_keys = copy.copy(list(vertex.attributes.keys()))
```

Comment:
if I'm not mistaken, then you don't need the `copy` here and can just iterate over the keys in the loop.